src/main/java/uk/ac/ebi/spot/gwas/curation/service/impl/SubmissionServiceImpl.java

```diff
@@ -176,4 +187,73 @@ public Submission   patchSubmission(SubmissionDto submissionDto, String submissi
         return submissionRepository.save(submission);
 
     }
+
+
+    public Map<String, SubmissionEnvelope> getSubmissions() {
+        Map<String, SubmissionEnvelope> submissionList = new TreeMap<>();
+        try {
+            int i = 0;
+            Map<String, Integer> params = new HashMap<>();
+            params.put("page", i);
+            log.info("Ingest uri is "+ restInteractionConfig.getIngestServiceUri()+restInteractionConfig.getSubmissionEnvelopeEndpoint());
+            DepositionSubmission[] submissions =
+                    restTemplate.getForObject(restInteractionConfig.getIngestServiceUri()+restInteractionConfig.getSubmissionEnvelopeEndpoint(), DepositionSubmission[].class, params);
+            Arrays.stream(submissions).forEach(s -> {
+                SubmissionEnvelope testSub = buildSubmission(s);
+                submissionList.put(testSub.getId(), testSub);
+            });
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Hi @sajo-ebi could you just properly log this, printing to staktrace isa security risk